Detections/SecurityEvent/AuditPolicyManipulation_using_auditpol.yaml

```diff
@@ -0,0 +1,85 @@
+id: 66276b14-32c5-4226-88e3-080dacc31ce1
+name: Audit policy manipulation using auditpol utility
+description: |
+   This detects attempt to manipulate audit policies using auditpol command.
+   This technique was seen in relation to Solorigate attack but the results can indicate potential  malicious activity used in different attacks.
+   Refer to auditpol syntax: https://docs.microsoft.com/windows-server/administration/windows-commands/auditpol  
```

Comment:
Include a reference to our M365D blog - https://www.microsoft.com/security/blog/2021/01/20/deep-dive-into-the-solorigate-second-stage-activation-from-sunburst-to-teardrop-and-raindrop/